sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -0,0 +1,933 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  WebResource,
+  ServiceClient,
+  ServiceClientOptions,
+  HttpOperationResponse,
+  signingPolicy,
+  logPolicy,
+  proxyPolicy,
+  RequestPolicyFactory,
+  URLBuilder,
+  ProxySettings
+} from "@azure/core-http";
+
+import { AtomXmlSerializer, executeAtomXmlOperation } from "./util/atomXmlHelper";
+
+import * as log from "./log";
+import { SasServiceClientCredentials } from "./util/sasServiceClientCredentials";
+import * as Constants from "./util/constants";
+
+import {
+  QueueResourceSerializer,
+  InternalQueueOptions,
+  QueueOptions,
+  buildQueueOptions,
+  Queue,
+  buildQueue
+} from "./serializers/queueResourceSerializer";
+import {
+  TopicResourceSerializer,
+  InternalTopicOptions,
+  TopicOptions,
+  buildTopicOptions,
+  Topic,
+  buildTopic
+} from "./serializers/topicResourceSerializer";
+import {
+  SubscriptionResourceSerializer,
+  InternalSubscriptionOptions,
+  SubscriptionOptions,
+  buildSubscriptionOptions,
+  Subscription,
+  buildSubscription
+} from "./serializers/subscriptionResourceSerializer";
+import {
+  RuleResourceSerializer,
+  InternalRuleOptions,
+  RuleOptions,
+  buildRuleOptions,
+  Rule,
+  buildRule
+} from "./serializers/ruleResourceSerializer";
+
+/**
+ * Request options for list<entity-type>() operations
+ */
+export interface ServiceBusAtomManagementClientOptions {
+  /**
+   * Count of entities to fetch.
+   */
```

Comment:
This doesn't look like the right comment.

Reply:
Fixed